git-gui/po/README

```diff
@@ -37,9 +37,9 @@ files whose names end with ".po".  Is there a file that has messages
 in your language?
 
 If you do not know what your language should be named, you need to find
-it.  This currently follows ISO 639-1 two letter codes:
+it.  This currently follows ISO 639-1 two-letter codes:
```

Comment:
We recently had a pretty disruptive change from "up-to-date" to "up to date". In this light, I don't think that "two-letter" is an improvement over "two letter".